UnityProject/Assets/Scripts/Managers/SubSceneManager/SubSceneManager.Client.cs

```diff
@@ -29,7 +32,7 @@ void MonitorServerSceneListOnClient()
 		}
 	}
 
-	IEnumerator LoadClientSubScene(SceneInfo sceneInfo)
+	IEnumerator LoadClientSubScene(SceneInfo sceneInfo, bool SynchronisingHandled = true)
```

Comment:
Except this one, what's up with the unused parameter?

Reply:
it should be used, I think in the original PR it wasn't